packages/extension-api-explorer/src/index.tsx

```diff
@@ -27,14 +27,14 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
 import { ExtensionProvider } from '@looker/extension-sdk-react/lib'
-import { ApiExplorerProvider } from './ApiExplorerProvider'
+import { ExtensionApiExplorer } from './ExtensionApiExplorer'
 
 window.addEventListener('DOMContentLoaded', () => {
   const root = document.createElement('div')
   document.body.appendChild(root)
   ReactDOM.render(
     <ExtensionProvider requiredLookerVersion=">7.10.0">
```

Comment:
We might need to bump the Looker version requirement later on once the specs are part of the HT build. Nothing to address in this PR, just a reminder for us.